cmd/snap/skills_common_test.go

```diff
@@ -0,0 +1,76 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main_test
+
+import (
+	. "gopkg.in/check.v1"
+
+	. "github.com/ubuntu-core/snappy/cmd/snap"
+)
+
+type AttributePairSuite struct{}
+
+var _ = Suite(&AttributePairSuite{})
+
+func (s *AttributePairSuite) TestUnmarshalFlagAttributePair(c *C) {
+	var ap AttributePair
+	// Typical
```

Comment:
This makes me think that some of those should be actual errors. I'll return here to make that happen.